gerrit-entrypoint.sh

```diff
@@ -9,6 +9,10 @@ set_secure_config() {
   gosu ${GERRIT_USER} git config -f "${GERRIT_SITE}/etc/secure.config" "$@"
 }
 
+if [ -n "${JAVA_HEAPLIMIT}" ]; then
+  JAVA_MEM_OPTIONS="-Xmx${JAVA_HEAPLIMIT}"
+fi
+
```

Comment:
This will be unnecessary if you specify the heap limits into the JAVA_OPTIONS directly like `JAVA_OPTIONS="-Xmx2g"`.